doc/rtd/topics/format.rst

```diff
@@ -38,29 +38,10 @@ Supported content-types:
 Helper script to generate mime messages
 ---------------------------------------
 
-.. code-block:: python
-
-   #!/usr/bin/python
-
-   import sys
-
-   from email.mime.multipart import MIMEMultipart
-   from email.mime.text import MIMEText
-
-   if len(sys.argv) == 1:
-       print("%s input-file:type ..." % (sys.argv[0]))
-       sys.exit(1)
-
-   combined_message = MIMEMultipart()
-   for i in sys.argv[1:]:
-       (filename, format_type) = i.split(":", 1)
-       with open(filename) as fh:
-           contents = fh.read()
-       sub_message = MIMEText(contents, format_type, sys.getdefaultencoding())
-       sub_message.add_header('Content-Disposition', 'attachment; filename="%s"' % (filename))
-       combined_message.attach(sub_message)
+The cloud-init codebase includes a helper script to generate MIME multi-part
+files: `make-mime.py`_
```

Comment:
Should we include an example usage?  I think we already so one user not immediately familiar with the syntax required (mime-type)|(path to file to include)

Reply:
The syntax is `<path>:<mime-type>` so I think we're now 3 for 3 (I also didn't get it right the first few times) as far as confusion goes. ;-)

Will add!

Reply:
Done.